src/ansys/mapdl/core/mapdl_extended.py

```diff
@@ -2212,10 +2219,21 @@ def load_table(self, name, array, var1="", var2="", var3="", csysid=""):
             )
 
         if array.ndim == 2:
+            # MAPDL considers the first row to be column header when col num > 2.
+            # When col header is not available duplicate the first row
+            if array.shape[1] > 2 and col_header == False:
+                array = np.vstack((array[0], array))
+                imax_val = array.shape[0] - 1
+            # When col header is available use col header
+            elif array.shape[1] > 2 and col_header == True:
+                imax_val = array.shape[0] - 1
+            else:
+                imax_val = array.shape[0]
```

Comment:
Just nitpicking, feel free to discard this suggestions:

```suggestion
            # MAPDL considers the first row to be column header when col num > 2.
            # When col header is not available duplicate the first row
            if array.shape[1] > 2:
                if col_header is False:
                    array = np.vstack((array[0], array))
                imax_val = array.shape[0] - 1
            else:
                imax_val = array.shape[0]
```